temperature/crud.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+import httpx
+from fastapi import HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from . import models
```

Comment:
Ensure that the `models` module contains the `Temperature` model, which is used for database operations related to temperature records.